ipn/ipnstate/ipnstate.go

```diff
@@ -522,7 +529,7 @@ table tbody tr:nth-child(even) td { background-color: #f5f5f5; }
 			}
 		}
 		var owner string
-		if up, ok := st.User[ps.UserID]; ok {
+		if up, ok := st.User[tailcfg.UserID(ps.UserID)]; ok {
```

Comment:
i think you should also make this map be keyed of `UserID` instead of `tailcfg.UserID`